src/python/pants/backend/terraform/lint/tffmt/tffmt.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+import logging
+import os
+import textwrap
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Dict
+
+from pants.backend.terraform.lint.fmt import TerraformFmtRequest
+from pants.backend.terraform.target_types import TerraformSources
+from pants.backend.terraform.tool import TerraformTool
+from pants.core.goals.fmt import FmtResult
+from pants.core.goals.lint import LintResult, LintResults
+from pants.core.util_rules import external_tool
+from pants.core.util_rules.external_tool import DownloadedExternalTool, ExternalToolRequest
+from pants.core.util_rules.source_files import SourceFiles, SourceFilesRequest
+from pants.engine.fs import Digest, MergeDigests
+from pants.engine.internals.selectors import Get, MultiGet
+from pants.engine.platform import Platform
+from pants.engine.process import FallibleProcessResult, Process, ProcessResult
+from pants.engine.rules import collect_rules, rule
+from pants.engine.target import FieldSet
+from pants.engine.unions import UnionRule
+from pants.option.subsystem import Subsystem
+from pants.util.logging import LogLevel
+from pants.util.strutil import pluralize
+
+logger = logging.getLogger(__name__)
+
+
+class TfFmtSubsystem(Subsystem):
+    options_scope = "tffmt"
```

Comment:
My previous comment about the options scope being more readable, e.g., `terraform-fmt`, still stands unless there is a stated reason not to.

Reply:
At this point, I will do that in an immediate follow-up. It will be easier to review that way.

Reply:
That's fine to do in an immediate followup. Thanks

Reply:
Given that there was a lint failure, it would be preferable to fix this in this PR now

Reply:
Fixed.